yarn.lock

```diff
@@ -2,6 +2,11 @@
 # yarn lockfile v1
 
 
+"@adraffy/ens-normalize@1.10.1":
```

Comment:
Not sure why there are changes on this file, package.json haven't changed.

Reply:
901fd40330213cecb5668b6621af67f2da91bd07